### ISSUE TITLE
different block color for 'suspicious' graph blocks in overview

### ISSUE THREAD

**Issue by mrexodia** [x64dbg/x64dbg#1316]
Blocks containing `call/jmp reg/[non-const mem]` should be highlighted in a different color (they are used in packers to jump to dynamically constructed code sections).

**Comment by Nukem9** [comment]
This won't work when debugging an exe with a large amount of virtual function calls. Every other block would be highlighted if debugging a game using DirectX/COM apis.

**Comment by mrexodia** [comment]
Yeah but it would still work fine. It just highlights blocks with virtual
function calls (and those are interesting).
